demo/component/BrushYDemo.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component, useState } from 'react';
+import { LineChart, Line, Brush, BrushY, XAxis, YAxis } from 'recharts';
+
+const data = [
+  { day: '1', value: 10 },
+  { day: '2', value: 20 },
+  { day: '3', value: 10 },
+  { day: '4', value: 30 },
+  { day: '5', value: 50 },
+  { day: '6', value: 10 },
+  { day: '7', value: 30 },
+  { day: '8', value: 20 },
+  { day: '9', value: 10 },
+  { day: '10', value: 70 },
+  { day: '11', value: 40 },
+  { day: '12', value: 20 },
+  { day: '13', value: 10 },
+  { day: '14', value: 10 },
+];
+
+function BrushYDemo() {
```

Comment:
We don't really use the demo project anymore, can you add this code to the storybook instead?

Reply:
Of course, as soon as I figure out how to do it :)

Reply:
Should be similar code luckily! You can add this to the Brush examples: storybook - Examples - Cartesian - Brush

Reply:
I'll add it as a new component as I couldn't figure out how to incorporate changes in the Brush component and have it rendered twice. I trust that the planned refactoring will enable this. Until then...